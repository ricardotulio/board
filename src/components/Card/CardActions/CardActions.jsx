import React from 'react'
import styles from './styles.module.css'

const CardActions = () => (
  <div className={styles.actions}>
    <a href='#'>edit</a>
    <a href='#'>delete</a>
  </div>
)

export default CardActions
