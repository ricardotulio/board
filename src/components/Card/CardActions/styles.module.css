.actions {
  border-top: 1px solid rgba(0, 0, 0, .1);
  text-align: right;
}

.actions a {
  margin-left: 5px;
}
