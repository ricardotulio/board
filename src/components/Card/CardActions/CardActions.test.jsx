import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import CardActions from './CardActions'

it('can render an action bar', () => {
  const { container } = render(<CardActions />)
  expect(container.firstChild).toMatchSnapshot()
})
