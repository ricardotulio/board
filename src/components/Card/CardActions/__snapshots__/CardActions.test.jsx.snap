// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render an action bar 1`] = `
<div
  class="actions"
>
  <a
    href="#"
  >
    edit
  </a>
  <a
    href="#"
  >
    delete
  </a>
</div>
`;
